src/zappy_gui/main.cpp

```diff
@@ -0,0 +1,28 @@
+/*
+** EPITECH PROJECT, 2022
+** ZAPPYMIRROR
+** File description:
+** main
+*/
+
+#include "data_game.h"
+#include "GameInfo.hpp"
+#include <exception>
+#include <iostream>
+#include <string>
+
+int main(int ac, char **av)
+{
+    if (ac < 3)
+        std::cerr << "Not enough arguments" << std::endl;
+    if (ac > 3)
+        std::cerr << "Too many arguments" << std::endl;
+    try {
+        std::string machine(av[2]);
+        gui::GameInfo info = gui::GameInfo(std::stoi(av[1]), machine);
+    } catch (std::exception const &e) {
```

Comment:
Create a custom error class will be very usefull for logging informations for the next step of the project. Think about it